data/Yashwanth.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "Yashwanth krishna",
+  "bio": "Trying to contribute to open source while learning. Learing in Public ",
+  "tags": ["Machine learning","DSA"],
+  "social": [
+    {
+      "icon": "Fatwitter",
+      "Url": "https://twitter.com/Yashwanthstwt",
+    },
+    {
+      "icon": "Falinkedin",
+      "url": "https://www.linkedin.com/in/pyashwanthkrishna/",
+    }
+  ],
+  
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Yashwanthstwt",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Gitub link",
+      "url": "https://github.com/yashwanth-3000",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Linkedin profile ",
+      "url": "https://www.linkedin.com/in/pyashwanthkrishna/"
+      "icon": "FaLinkedin"
+    },
```

```suggestion
    }
```
This is a trailing comma which isnot required

---

have changed this user name and comma is that fine?

---

> have changed this user name and comma is that fine?

No the filename must be yashwanth-3000.json It must have the .json file extension.